### ISSUE TITLE
Batch Normalization moving_mean parameter seems to ignore the initializer.

### ISSUE THREAD

**Issue by benjamin-lucas** [keras-team/keras#13117]
I have a model with multiple convolutional layers with BN applied before the activation. I've noticed that after training on one batch of data, the moving_average parameter always takes the values of that batch (almost like an intialization), rather than updating the initial value with the new batch. This means that regardless the initial values, after training on one batch, the moving_average is always the same.

To test, this I added an output after the first convolutional layer and calculated the mean of each filter... and it is clear that the mean of each filter (labelled Batch Averages below) are the value of the moving_average after training on one batch.

**Output**
Initial Values set to those of a pretrained model:

Intial moving_average values:  [-1.3024728  -0.19747962  0.25722674  0.19224428 -2.536243    0.09171127
 -0.7711318  -0.3154904  -0.424648   -0.72937334 -1.1128609  -0.7815677
  0.37255728  0.22089313 -0.01141051 -0.40124747 -0.29431173 -0.46932122
 -0.20156854 -0.96680003  0.02309855  0.29308507 -0.52985156 -0.21760845
  1.7136794  -0.41971454  1.0172414  -0.56221694 -1.3398308  -0.30157936
  0.3720148  -0.03523164 -0.5264422   0.90892845 -0.32599527  0.13658263
 -0.4408616  -0.23081435 -0.4587749  -0.09867792 -0.21861517 -0.03027776
 -0.21881467 -0.5239249  -0.09058609 -0.50434935 -0.8656431  -0.3341593
 -0.05375587 -0.5310102   0.06130832  0.02043517  0.16636467 -0.5852665
 -1.1550901   0.15761244  0.04864887  1.2416579  -0.04080978 -0.20459446
 -1.4095263  -0.624391    2.7961416  -0.25953373]
Batch Averages:  [-1.1140723  -0.16110186  0.332009    0.19606787 -3.5637982  -0.14254743
 -0.68367666 -0.967366   -0.15435079 -0.7180207  -2.3669355  -0.5456788
  0.34254047  0.27768376  0.2972685  -0.55385596 -0.23607543 -0.6717377
 -0.24737652 -0.4900981   0.07956652  0.37054494 -0.54882646 -0.21973458
  2.1991184  -0.3650401   1.2412876  -1.0896026  -1.2301126  -0.9274771
  0.22961889  0.17862722 -0.8987975   1.0209091  -0.21280843  0.22950041
 -0.33305508 -0.5612238  -0.61949444  0.05236189 -0.3216586  -0.3393619
 -0.06517737 -0.45107856 -0.14314142 -0.49852288 -0.6803146  -0.78803325
  0.21186857 -0.3735427   0.07193005 -0.19647199  0.14294447 -0.9254668
 -1.5565863   0.25526142 -0.02683147  1.3166173  -0.22105207 -0.0635791
 -1.3188571  -1.0641339   3.5579991  -0.20571914]
Moving_average values after training on one batch:  [-1.1140734  -0.16110201  0.3320093   0.19606806 -3.5638015  -0.14254756
 -0.6836773  -0.9673668  -0.15435094 -0.71802133 -2.3669376  -0.54567933
  0.3425408   0.27768403  0.29726878 -0.5538565  -0.23607564 -0.67173827
 -0.24737674 -0.49009854  0.07956659  0.3705453  -0.54882693 -0.21973479
  2.1991205  -0.36504042  1.2412888  -1.0896035  -1.2301137  -0.927478
  0.2296191   0.17862739 -0.8987984   1.02091    -0.21280862  0.22950062
 -0.33305538 -0.56122434 -0.619495    0.05236194 -0.3216589  -0.33936223
 -0.06517743 -0.45107898 -0.14314155 -0.49852335 -0.6803152  -0.78803396
  0.21186876 -0.37354302  0.07193012 -0.19647217  0.1429446  -0.92546767
 -1.5565877   0.25526166 -0.02683149  1.3166184  -0.22105227 -0.06357916
 -1.3188584  -1.0641348   3.5580025  -0.20571934]

Initialized with zeros

Intial moving_average values:  [0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.
 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.
 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
Batch Averages:  [-1.1140723  -0.16110186  0.332009    0.19606787 -3.5637982  -0.14254743
 -0.68367666 -0.967366   -0.15435079 -0.7180207  -2.3669355  -0.5456788
  0.34254047  0.27768376  0.2972685  -0.55385596 -0.23607543 -0.6717377
 -0.24737652 -0.4900981   0.07956652  0.37054494 -0.54882646 -0.21973458
  2.1991184  -0.3650401   1.2412876  -1.0896026  -1.2301126  -0.9274771
  0.22961889  0.17862722 -0.8987975   1.0209091  -0.21280843  0.22950041
 -0.33305508 -0.5612238  -0.61949444  0.05236189 -0.3216586  -0.3393619
 -0.06517737 -0.45107856 -0.14314142 -0.49852288 -0.6803146  -0.78803325
  0.21186857 -0.3735427   0.07193005 -0.19647199  0.14294447 -0.9254668
 -1.5565863   0.25526142 -0.02683147  1.3166173  -0.22105207 -0.0635791
 -1.3188571  -1.0641339   3.5579991  -0.20571914]
Moving_average values after training on one batch:  [-1.1140734  -0.16110201  0.3320093   0.19606806 -3.5638015  -0.14254756
 -0.6836773  -0.9673669  -0.15435092 -0.71802133 -2.3669376  -0.54567933
  0.3425408   0.27768403  0.29726878 -0.5538565  -0.23607564 -0.67173827
 -0.24737674 -0.49009854  0.07956659  0.3705453  -0.54882693 -0.21973479
  2.1991205  -0.36504042  1.2412888  -1.0896035  -1.2301137  -0.927478
  0.2296191   0.17862739 -0.8987984   1.02091    -0.21280862  0.22950062
 -0.33305538 -0.56122434 -0.619495    0.05236193 -0.3216589  -0.33936223
 -0.06517743 -0.45107898 -0.14314155 -0.49852335 -0.6803152  -0.78803396
  0.21186878 -0.37354302  0.07193012 -0.19647217  0.1429446  -0.92546767
 -1.5565877   0.25526166 -0.02683149  1.3166184  -0.22105227 -0.06357916
 -1.3188584  -1.0641348   3.5580025  -0.20571934]

I cannot find how the initializer works for the moving_average but I assume this behavior is not intended as otherwise there'd be no need for the initializer at all.

The same occurs with the moving_variance parameter.

**Comment by fpetitjean** [comment]
This is actually not a support request but a bug
